tools/mo/openvino/tools/mo/moc_frontend/pytorch_frontend_utils.py

```diff
@@ -131,6 +131,7 @@ def convert_pytorch_to_onnx(model, input_shape, opset_version, example_inputs, o
     torch.onnx.export(model,
                       inputs,
                       model_onnx,
+                      operator_export_type=torch.onnx.OperatorExportTypes.ONNX_FALLTHROUGH,
                       **additional_params)
```

Comment:
not a blocker for this PR but we need to consider `try-except` to properly handle possible exceptions.
@popovaan, can you create separate PR please for this?

Reply:
Yes, I can add `try-except` is separate PR